### PR TITLE
chore: repo updates

### PR DESCRIPTION
**Pull request by CloudBeard**
Updated repo to become a lula first class repo.

### CODE REVIEW

**Review by brandtkeller — CHANGES_REQUESTED** [review]
Release please and other configuration for renovate/ pipelines/ templates etc looks good. 

I don't believe we need any golang related content - to include `goreleaser` and I'd also vet how necessary the sbom action tool is.

Are we able to down-select and remove unnecessary content?

**Comment by CloudBeard** [comment]
The SBOM action is needed for goreleaser. removed goreleaser and all of those dependencies.

File: docs/community-and-contribution/release-process.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# Lula-Action Release
+
+This document provides guidance on how to create Lula-Action releases, address release issues, and other helpful tips.
+
+## Creating releases
+
+This project uses [goreleaser](https://github.com/goreleaser/goreleaser-action) for releasing binaries and [release-please](https://github.com/marketplace/actions/release-please-action) for creating release PR's.
```
Comment (brandtkeller) [inline docs/community-and-contribution/release-process.md:7]: mention of go-releaser needs removal

File: docs/community-and-contribution/release-process.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+- Manual approach: Find the impacted release, edit the release notes, and put this warning at the top:
+
+```md
+>[!WARNING]
+>PLEASE USE A NEWER VERSION (there are known issues with this release)
+```
+
+#### Other issues and helpful tips
+
+- Confirm that the goreleaser configuration is valid by using the [goreleaser cli](https://goreleaser.com/cmd/goreleaser_check/?h=valid)
```
Comment (brandtkeller) [inline docs/community-and-contribution/release-process.md:45]: mention of go-releaser needs removal

**Review by brandtkeller — CHANGES_REQUESTED** [review]
Updated the repo maintainers to include `lula-dev` such that the codeowners file is now valid. Few mentions of go-releaser to remove and update accordingly otherwise this looks on track.

File: README.md [hunk 3]
```diff
@@ -7,12 +7,13 @@ Make [Lula](https://github.com/defenseunicorns/lula) available to your GitHub Ac
 ### Linting OSCAL file against schema
 ```yaml
 - name: Setup Lula
-  uses: defenseunicorns/lula/setup@v1
+  uses: defenseunicorns/lula-action/setup@v1
```
Comment (brandtkeller) [inline README.md:10]: can you change the version to main on all of these real quick? Until we sort out which version we are going to use.

**Review by brandtkeller — APPROVED** [review]

**Pull request merged**